tc/sgx/trusted_worker_manager/enclave/kme/CMakeLists.txt

```diff
@@ -0,0 +1,76 @@
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+CMAKE_MINIMUM_REQUIRED(VERSION 3.2 FATAL_ERROR)
+PROJECT(libavalon-kme-enclave  CXX)
+
+INCLUDE(../CMakeVariables.txt)
+
+# XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+# ADDITIONAL LIBRARIES
+# XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+
+SET(TCF_TOP_DIR "${CMAKE_CURRENT_SOURCE_DIR}/../../../../../")
+SET(TCF_CORE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/../../../")
```

Comment:
You could use `PARENT_SCOPE` for cmake variables to keep the variable scope in parent cmakefile or you could actually move `INCLUDE(../CMakeVariables.txt)` after setting `TCF_TOP_DIR` and `TCF_CORE_DIR`

Reply:
Moved INCLUDE(../CMakeVariables.txt) after setting TCF_TOP_DIR and TCF_CORE_DIR